src/antsibull_changelog/update_galaxy.py

```diff
@@ -0,0 +1,26 @@
+# Author: Sagar Paul <paul.sagar@yahoo.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2023, Ansible Project
+
+"""
+Update the galaxy the yaml file.
+"""
+
+from __future__ import annotations
+
+from .errors import ChangelogError
+from .yaml import load_yaml, store_yaml
+
+
+def update_galaxy(galaxy_path: str, version: str) -> None:
+    """
+    Load and update galaxy.yaml file.
+    """
+    if galaxy_path:
+        config = load_yaml(galaxy_path)
+        config["version"] = str(version)
```

Comment:
The function has `version: str` are an argument. There's no need to coerce it explicitly.

```suggestion
        config["version"] = version
```